core/src/test/java/org/frankframework/util/TransformerPoolNamespaceUnawarenessTest.java

```diff
@@ -12,31 +12,16 @@
 
 public class TransformerPoolNamespaceUnawarenessTest {
 
-	public static String NAMESPACED_INPUT_MESSAGE="<root><sub xmlns=\"http://dummy\">+</sub><sub>-</sub></root>";
-	public static String NAMESPACELESS_XPATH="/root/sub";
-	public static String NAMESPACELESS_STYLESHEET="/Util/TransformerPool/NamespacelessStylesheet.xsl";
-	public static String NAMESPACE_INSENSITIVE_RESULT= "+ -";
-	public static String NAMESPACE_COMPLIANT_RESULT= "-";
-	public static String NAMESPACE_INSENSITIVE_FIRST_RESULT= "+";
-	public static String NO_MATCH_AT_ALL= "";
+	public static final String NAMESPACED_INPUT_MESSAGE="<root><sub xmlns=\"http://dummy\">+</sub><sub>-</sub></root>";
+	public static final String NAMESPACELESS_XPATH="/root/sub";
+	public static final String NAMESPACELESS_STYLESHEET="/Util/TransformerPool/NamespacelessStylesheet.xsl";
+	public static final String NAMESPACE_INSENSITIVE_RESULT= "+ -";
+	public static final String NAMESPACE_COMPLIANT_RESULT= "-";
+	public static final String NAMESPACE_INSENSITIVE_FIRST_RESULT= "+";
 
-
-	public String XPATH_0_AND_2_RESULT_7_0 = NAMESPACE_COMPLIANT_RESULT;
-	public String XPATH_0_AND_2_RESULT_7_5678 = NAMESPACE_INSENSITIVE_RESULT;
-
-	public String xpath_0_and_2_result = XPATH_0_AND_2_RESULT_7_5678;
-
-
-	public String STYLESHEET_AUTO_UNAWARE_RESULT_7_0= NAMESPACE_COMPLIANT_RESULT;
-	public String STYLESHEET_AUTO_UNAWARE_RESULT_7_5678= NAMESPACE_INSENSITIVE_FIRST_RESULT;
-
-	public String stylesheet_auto_unaware_result = STYLESHEET_AUTO_UNAWARE_RESULT_7_5678;
-
-
-	public String XSLT1_UNAWARE_RESULT_7_0= NAMESPACE_INSENSITIVE_FIRST_RESULT;
-	public String XSLT1_UNAWARE_RESULT_7_5678= NAMESPACE_INSENSITIVE_FIRST_RESULT;
```

Comment:
Poeh, tja dit is een van de redenen dat niemand wil upgraden, in elke versie was het anders 😢